src/components/CardList.jsx

```diff
@@ -1,36 +1,14 @@
 import styled from 'styled-components';
-import { useEffect, useState } from "react";
-
-function CardList() {
-
-  const apiURL="https://rickandmortyapi.com/api/character";
-  const [characters, setCharacters] = useState([]);
-  const [showMore, setShowMore] = useState(false);
-
-  const Characters =() =>{
-    fetch(apiURL).then((response) => response.json()).then((data) => setCharacters(data.results))
-  }
-
-
-  useEffect(() => {
-    Characters();
-  }, []);
-
-  const addInfo = () => {
-    setShowMore(!showMore);
-    console.log("hello"); 
-  }
 
+function CardList({ characters, showMore, toggleInfo}) {
   return (
-  <CardListStyled>
-   {characters.map(({ name, image, id, status, species, gender }) => {
-     return (
-       <CardContent key={id}>
-          <img src={image}>
-          </img>
-         <CharacterName>{name}</CharacterName>
-         <button onClick={addInfo}>Show more</button>
-         <div>
+    <CardListStyled>
+      {characters.map(({ name, image, id, species, gender, status }) => (
+        <CardContent key={id}>
```

Comment:
There should be some kind of link to the details page within the "Card component"